src/Footer/Footer.scss

```diff
@@ -0,0 +1,8 @@
+﻿footer {
```

Comment:
I'd still prefer that this applied only our one component.

At the moment, this will apply to all footers on the page. 
That's probably ok, because there will probably only be 1 footer on the page, but it still feels a little bit wrong! :) 

We could do this by giving this footer a class, and then only applying the styles to exactly that class.